background/get-userscripts.js

```diff
@@ -262,6 +262,29 @@ function userscriptMatches(data, scriptOrStyle, addonId) {
 }
 
 function urlMatchesPattern(pattern, url) {
+  switch (pattern) {
+    case "all":
+      pattern = "https://scratch.mit.edu/*";
+      break;
+    case "projects":
+      pattern = "https://scratch.mit.edu/projects/*";
+      break;
+    case "profiles":
+      pattern = "https://scratch.mit.edu/users/*/";
+      break;
+    case "studio":
+      pattern = "https://scratch.mit.edu/studios/*";
+      break;
+    case "forums":
+      pattern = "https://scratch.mit.edu/studios/*";
+      break;
+    case "forums_topic":
```

Comment:
Shouldn't these be camelCase?

Reply:
For what reason?

Reply:
@TheColaber Everything in `addon.json` is currently camel case.